src/components/App.css

```diff
@@ -51,4 +51,15 @@ html, body, #root {
   padding: 0 24px;
   cursor: pointer;
   transition: color .3s;
+}
+
+.item-title {
```

Comment:
Choosing blue as a headline color is probably unfortunate. It made me think it was a link. Blue is used as a link color in the remainder of the project.

Reply:
Ahh - I see now why you did that. https://registry-demo.gbif-dev.org/web/#/organization/74a7e341-76f1-4816-a449-ce6482df9e16/endpoint

That is a mistake in the existing console. It shows an a-tag but not the href. Tim just forgot that when implementing it. It should of course link to the the actual url instead :)

Reply:
I added links everywhere where I saw a possibility to do that. I'll check titles again during code review fixes. Thank you!